streams/src/test/java/org/apache/kafka/streams/state/internals/MergedSortedCacheWrappedWindowStoreIteratorTest.java

```diff
@@ -85,4 +85,18 @@ public void shouldPeekNextKey() throws Exception {
         assertThat(iterator.peekNextKey(), equalTo(10L));
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldPeekNextKey2() throws Exception {
```

Comment:
Could we just replace the `shouldPeekNextKey` with this one since its coverage seems to be the "superset" of the old one?